src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature

```diff
@@ -131,3 +131,86 @@ Scenario: Additional Services - Select service recipient - Go back
 	And the User has selected a Additional Service price
 	When the User chooses to go back
 	Then all the available prices for that Additional Service are presented
+
+Scenario: Additional Services - Flat price with variable order type selected
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	Then they are presented with the Additional Service edit form for flat list price
+	And the form contains one item
+	And the item contains an input for the price
+	And the item contains a unit of order
+	And the item contains an input for the quantity
+	And the price input is autopopulated with the list price for the flat list price selected
+	And the delete button is disabled
+	And the save button is enabled
+
+Scenario: Additional Services - Mandatory data missing
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And the User chooses to save
+	Then the Additional Service is not saved
+	And the reason is displayed
+
+Scenario: Additional Services - Data exceeds the maximum length
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And the quantity is above the max value
+	And the User chooses to save
+	Then the Additional Service is not saved 
+	And the reason is displayed
+
+Scenario: Additional Services - Validation Error Message Anchors
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And the User chooses to save
+	And the user selects an error link in the Error Summary
+	Then they will be navigated to the relevant part of the page
+
+Scenario: Additional Services - All data are valid
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
```

Comment:
I think you should take a look at the catalogue solutions feature file and see how the Given steps can be aggregated into a new step Given I'm on the edit price form